mftf/2.3/test/action-groups.md

```diff
@@ -178,6 +178,57 @@ MFTF resolves `{{myCustomEntity.field1}}` the same as it would in a `selector` o
     <argument name="relevantString" value="{{myCustomEntity.field1}}"/>
 </actionGroup>
 ```
+## Optimizing action group structures
+
+Structuring an action group properly increases code reusability and readability. 
+
+Starting with an action group like:
```

Comment:
We try to avoid using "like" (though fully acceptable); can substitute with "such as" or similar. It's per writer preference really. I know @hguthrie would strip it out 😄

Reply:
Fixed.

Reply:
Like, totally. OMG.

Reply:
We can sass after merging?!